testing/_py/test_local.py

```diff
@@ -209,7 +209,7 @@ def test_visit_norecurse(self, path1):
 
     @pytest.mark.parametrize(
         "fil",
-        ["*dir", "*dir", pytest.mark.skip("sys.version_info < (3,6)")(b"*dir")],
+        ["*dir", "*dir", pytest.mark.skipif("sys.version_info < (3,10)")(b"*dir")],
```

Comment:
Perhaps, it'd be better to unquote the check and add a `reason=`?
```suggestion
        ["*dir", "*dir", pytest.mark.skipif(sys.version_info < (3, 10))(b"*dir")],
```